src/ringct/rctOps.cpp

```diff
@@ -362,14 +374,14 @@ namespace rct {
         {
             return it->commitment;
         }
-        ge_p3 res_ge_p3 = G_GE_P3;
+        ge_p3 res_ge_p3 = G_GE_P3();
         static_assert(sizeof(xmr_amount) * 8 == H_TABLE_SIZE, "unexpected size of h table");
         for (size_t i = 0; i < H_TABLE_SIZE; ++i)
         {
             if (amount & (1UL << i))
```

Comment:
Make sure to add https://github.com/j-berman/monero/commit/a3cf0d708146b0c21a940f6fd64dc9abb231022a here.